packages/calypso-build/webpack/sass.js

```diff
@@ -36,10 +37,17 @@ module.exports.loader = ( { includePaths, prelude, postCssOptions } ) => ( {
 		{
 			loader: require.resolve( 'sass-loader' ),
```

Comment:
Also, have we tried parallelizing the sass compilation? Looking at `thread-loader` in the codebase it seems like we only use it for transpilation currently.

Reply:
I did a sloppy attempt last night. But I should probably try again with fresher eyes.